### PR TITLE
fix(mdjs-viewer): fix styling mdjs-viewer

### PR DESCRIPTION
**Pull request by gerjanvangeest**
## What I did

1. Fix styling in darkmode
2. Make [slot="content"] selector more specific

### CODE REVIEW

File: docs/_assets/theme.css [hunk 1]
```diff
@@ -45,6 +48,9 @@ html {
     --rocket-search-highlight-color: #41ffb0;
     --rocket-search-hover-background-color: #6b717a;
     --rocket-search-fill-color: #fff;
+    --primary-text-inverse-color: #2c3e50;
```
Comment (bennypowers) [inline docs/_assets/theme.css:51]: @gerjanvangeest  Can you add you theme changes to `@rocket/launch` as well, and bump that package?

**Review by bennypowers — CHANGES_REQUESTED** [review]
Here's a side-by-side comparison of the changes as of this comment:

Screen Shot 2021-08-10 at 15 55 32

**Review by bennypowers — APPROVED** [review]
LGTM. @daKmoR ?

**Comment by daKmoR** [comment]
thanks for the fix 🙇‍♂️  and thx benny for reviewing it 🙇‍♂️

**Pull request merged**